examples/question-answering/README.md

```diff
@@ -53,7 +53,8 @@ python run_qa.py \
   --doc_stride 128 \
   --output_dir ./output/debug_squad/ \
   --use_habana \
-  --use_lazy_mode
+  --use_lazy_mode \
+  --throughput_warmup_steps 2
```

Comment:
is this througput_warmup_steps configurable since for 8x it may take more than 2

Reply:
Yeah you can give it any value. I put 2 in the examples because that corresponds to what I observed in practice, but this could be `--throughput_warmup_steps N` with N a positive integer.